odoo/tools/misc.py

```diff
@@ -1363,6 +1364,28 @@ def format_duration(value):
     return '%02d:%02d' % (hours, minutes)
 
 
+def format_float(env, value, precision_rounding=None, uom=None, lang_code=False):
```

Comment:
Could be great to have a docstring and some tests, especially if you want it to land in stable